Src/BootCamp.Chapter/InteractiveMenu.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class InteractiveMenu
+    {
+        public int Build(string[] options)
+        {
+            var index = 0;
+            Console.CursorVisible = false;
+            do
+            {
+                Console.Clear();
+                Console.WriteLine("Welcome to Password Manager.");
+                Console.WriteLine("Select an option.");
+                Console.WriteLine("==================================================");
+                
+                for (var i = 0; i < options.Length; i++)
+                {
+                    if (index == i) Console.BackgroundColor = ConsoleColor.Blue;
```

Comment:
Be more descriptive in your naming. Instead of index, write selectOptionNumber or similar.